gapii/cc/vulkan_extras.cpp

```diff
@@ -876,16 +875,11 @@ void VulkanSpy::SpyOverride_vkDestroyInstance(
 uint32_t VulkanSpy::SpyOverride_vkCreateBuffer(
     CallObserver*, VkDevice device, const VkBufferCreateInfo* pCreateInfo,
     const VkAllocationCallbacks* pAllocator, VkBuffer* pBuffer) {
-  if (is_suspended()) {
-    VkBufferCreateInfo override_create_info = *pCreateInfo;
-    override_create_info.musage |=
-        VkBufferUsageFlagBits::VK_BUFFER_USAGE_TRANSFER_SRC_BIT;
-    return mImports.mVkDeviceFunctions[device].vkCreateBuffer(
-        device, &override_create_info, pAllocator, pBuffer);
-  } else {
-    return mImports.mVkDeviceFunctions[device].vkCreateBuffer(
-        device, pCreateInfo, pAllocator, pBuffer);
-  }
+  VkBufferCreateInfo override_create_info = *pCreateInfo;
```

Comment:
@AWoloszyn I changed the behaviour here to always add TRANSFER_SRC usage, whether or not we are suspended. I need this bit to be set at least for all external buffers, but it seemed simpler to just enable it all the time. This mirrors the behaviour in `SpyOverride_vkCreateImage` (where there is no `is_suspended` check). If you think it is important to maintain the check, I can change this to only enable TRANSFER_SRC for external buffers.

Reply:
I think that is fine. (Also more consistent). This change was originally to be less invasive, but I expect this to matter not at all for buffers, so feel free to make the change.